docs/resources/kafka_cluster.md

```diff
@@ -16,14 +16,26 @@ Manages a **Kafka Cluster** on IonosCloud.
 This resource will create an operational Kafka Cluster. After this section completes, the provisioner can be called.
 
 ```hcl
+resource "ionoscloud_datacenter" "datacenter" {
```

Comment:
I propose these be left out in favor of https://github.com/ionos-cloud/terraform-provider-ionoscloud/pull/638#pullrequestreview-2234955964

Reply:
agreed